packages/frontend/Dockerfile

```diff
@@ -3,10 +3,14 @@
 # build stage
 FROM node:16.13-bullseye-slim as build-stage
 
+WORKDIR /opt/objectloader
```

Comment:
Are the changes in both dockerfiles correct? Also, are these really the only ones that need updating? The root Dockerfile also seems to install the viewer (and objectloader by proxy)